src/OrchardCore.Themes/TheTheme/Views/LoginMenu.cshtml

```diff
@@ -14,7 +14,7 @@
 
 @await RenderSectionAsync("NavbarTop", required: false)
 
-<ul class="navbar-nav ms-auto">
```

Comment:
Very interesting here that `ms-auto` apply only `margin-left: auto !important;` and does not change. I think this may be a bug in bootstrap.

Reply:
I expect that, at the beginning I though it's a class added by us

Reply:
I expect that, at the beginning I though it's a class added by us

Reply:
I expect that, at the beginning I though it's a class added by us

Reply:
`ms-auto` is a bootstrap utility. you may want to report it there. If this is by design it should have been called `margin-left-auto` not `ms-auto` ms stands for `margin-start` start mean left on RTL and right on LTR.

Reply:
We usually covert from LTR to RTL using gulp, but I'm not sure why `ms-auto` has been used here may be it fixes a bug and open another bug in RTL :)